Sources/SwiftPlot/GraphLayout.swift

```diff
@@ -383,6 +400,10 @@ extension HasGraphLayout {
         return []
     }
     
+    public var annotations: [Annotation] {
+        return []
+    }
+
```

Comment:
This makes the annotations a read only variable as you have no setter. Instead you should have:
```swift
public var annotations: [Annotation] {
        get { layout.annotations }
        set { layout.annotations = newValue }
    }
```